Nucleation_model.py

```diff
@@ -0,0 +1,89 @@
+#%%
+import datetime as dt
+import shutil
+import math as m
+import numpy as np
+import pandas as pd
+import random as ran
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+from scipy.integrate import solve_ivp
+#%%
+"""=========================== Constants ==========================="""
+
+k_B = 1.380649E-23 #J K-1 // Boltzmann's constant
+ep_0 = 8.854E-12  #F m-1 // electric permittivity of a vacuum
+pi = 3.141529
```

Comment:
the `math` package includes pi, so you can just use `m.pi` for greater precision.